operator/internal/controller/redpanda/redpanda_controller_test.go

```diff
@@ -455,6 +456,58 @@ func (s *RedpandaControllerSuite) TestClusterSettings() {
 	s.deleteAndWait(rp)
 }
 
+func (s *RedpandaControllerSuite) TestLicenseReal() {
+	license := os.Getenv("REDPANDA_SAMPLE_LICENSE")
+	if license == "" {
+		s.T().Skip("License environment variable is not provided")
+	}
+
+	rp := s.minimalRP()
+	rp.Spec.ClusterSpec.Statefulset.PodTemplate = &redpandav1alpha2.PodTemplate{
```

Comment:
For good measure, should we include a cluster config that's gated on the license? I think auto rebalancing is a good option here 🤔